state/spaces.go

```diff
@@ -90,6 +90,49 @@ func (s *Space) Subnets() ([]*Subnet, error) {
 	return results, nil
 }
 
+// Applications returns the applications associated with the Space.
+// Worst case runtime would be (EndpointXBindings) collections
+func (s *Space) Applications() ([]string, error) {
```

Comment:
I would rather push this logic into the API facade state shim or the facade itself. 

We really want to push back hard on adding logic to state which couples these modules.